### PR TITLE
PRS Sebhats relatives

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
Ras Sebhat's relatives according to mss of Dabra Maeso

### CODE REVIEW

File: new/PRS13292WaldaGabreel.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Walda Gabrəʾel</persName>
+                    <faith type="EOTC"/>
+                    <occupation type="political"/>
+                    <nationality type="Ethiopia"/>
+                    <nationality type="Ethiopia"/>
+                    <floruit notBefore="1850" notAfter="1950"></floruit>
+                </person>
+                <listRelation>
+                    <relation name="snap:SonOf" active="PRS13292WaldaGabreel" passive="PRS8668SebhatA" cert="medium"/>
+                    <relation name="snap:SonOf" active="PRS13292WaldaGabreel" passive="ESmy003"/>
```
Comment (eu-genia) [inline new/PRS13292WaldaGabreel.xml:59]: hasOwned, not SonOf

File: new/PRS13293ArayaSellase.xml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+                    <faith type="EOTC"/>
+                    <occupation type="political"/>
+                    <nationality type="Ethiopia"/>
+                    <birth/>
+                    <death/>
+                    <floruit/>
+                </person>
+                <listRelation>
+                    <relation name="snap:SonOf" active="PRS13293ArayaSellase" passive="PRS8668SebhatA" cert="medium"/>
+                    <relation name="snap:SonOf" active="PRS13293ArayaSellase" passive="ESmy003"/>
```
Comment (eu-genia) [inline new/PRS13293ArayaSellase.xml:60]: hasOwned

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/PRS13292WaldaGabreel.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply new/PRS13292WaldaGabreel.xml:59]: ok

File: new/PRS13293ArayaSellase.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply new/PRS13293ArayaSellase.xml:60]: ok

**Review by eu-genia — APPROVED** [review]

**Pull request merged**